src/models/parallelCoordinates.js

```diff
@@ -38,11 +38,24 @@ nv.models.parallelCoordinates = function() {
             // Setup Scales
             x.rangePoints([0, availableWidth], 1).domain(dimensionNames);
 
+            var onlyNanValues = {};
             // Extract the list of dimensions and create a scale for each.
             dimensionNames.forEach(function(d) {
+                var extent = d3.extent(data, function(p) { return +p[d]; });
```

Comment:
I know it was already calling the variable d, but can we change it to i or index or something to correctly identify it when glancing at the code? d is typically the series data object when used elsewhere...